test/checks/aria/no-implicit-explicit-label.js

```diff
@@ -0,0 +1,45 @@
+describe('no-implicit-explicit-label', function() {
+	'use strict';
+
+	var fixture = document.getElementById('fixture');
+	var queryFixture = axe.testUtils.queryFixture;
+	var check = checks['no-implicit-explicit-label'];
+	var checkContext = axe.testUtils.MockCheckContext();
+
+	afterEach(function() {
+		fixture.innerHTML = '';
+		checkContext.reset();
+	});
+
+	it('returns undefined when there is no label text or accessible text', function() {
+		var vNode = queryFixture(
+			'<div id="target" role="searchbox" contenteditable="true"></div>'
+		);
+		var actual = check.evaluate.call(checkContext, vNode.actualNode, {}, vNode);
+		assert.isUndefined(actual);
```

Comment:
This should be false. We don't want people looking at labels when there are none.